Makefile

```diff
@@ -48,6 +48,7 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
+	pip-compile --upgrade -o requirements/django.txt requirements/django.in
```

Comment:
This should go before the line that generates `test.txt`, since the output of this command is used for that.

Reply:
Updated the order of commands.